::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
: Copyright (c) 2019 Mirantis Inc., Enea AB and others.
:
: All rights reserved. This program and the accompanying materials
: are made available under the terms of the Apache License, Version 2.0
: which accompanies this distribution, and is available at
: http://www.apache.org/licenses/LICENSE-2.0
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Tue, 20 Nov 2018 23:16:44 +0200
Subject: [PATCH] reclass: Set ignore_overwritten_missing_references

The recent changes in `reclass` 1.6.x allow configuring this new
option; however the default for it is broken - see [1].

[1] https://github.com/salt-formulas/reclass/issues/77

Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 files/reclass/reclass-config.yml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/files/reclass/reclass-config.yml b/files/reclass/reclass-config.yml
index 00aba49..f4da04c 100644
--- a/files/reclass/reclass-config.yml
+++ b/files/reclass/reclass-config.yml
@@ -3,6 +3,7 @@ inventory_base_uri: /srv/salt/reclass
 pretty_print: True
 output: yaml

+ignore_overwritten_missing_references: True
 ignore_class_notfound: True
 ignore_class_regexp:
 - 'service.*'
